gcc/c-family/c.opt

```diff
@@ -1964,6 +1964,10 @@ fcontracts-nonattr-client-contracts=
 C++ Joined RejectNegative Enum(client_contract_check) Var(flag_contract_nonattr_client_check) Init (0)
 -fcontracts-nonattr-client-check=[none|pre|all] Select which contracts will be checked on the client side for non virtual functions
 
+fcontracts-nonattr-def-contracts=
+C++ Var(flag_contracts_nonattr_def_contracts) Enum(on_off) Joined Init(1) RejectNegative
+-fcontracts-nonattr-def-contract=[on|off]	Enable or disable contract checks on the definition side for all functions (default on).
```

Comment:
This is a typo? That should be
-fcontract-nonattr-def-contracts=[on|off]

I also think the option should just be spelled -fcontract-nonattr-definition-contracts, because "def" has ambiguous meanings.

Reply:
renamed